src/ui/components/DayInput.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FC, useState } from 'react';
+import { DayPicker } from 'react-day-picker';
+import { WhpptIcon } from './Icon';
+import { WhpptInput } from './Input';
+
+export type WhpptDayInputProps = {
+  date: Date;
+  onChange: (value: any) => void;
+};
+
+export const WhpptDayInput: FC<WhpptDayInputProps> = ({ date, onChange }) => {
+  const [showCalendar, setShowCalendar] = useState(false);
+
+  return (
+    <div className="whppt-gallery__day-picker__container">
+      <div className="whppt-image-editor-panel__date-picker-input" onClick={() => setShowCalendar(!showCalendar)}>
+        <WhpptInput
+          id="date"
+          label="Date"
+          info=""
+          error=""
+          type="text"
+          value={date ? new Date(date).toLocaleDateString('en-GB') : new Date().toLocaleDateString('en-US')}
```

Comment:
What decides on en-GB or en-US. Could we possibly leave it off and have the browser show its default?